.github/workflows/helm-test.yml

```diff
@@ -2,6 +2,8 @@ name: Otel Agent Helm Install Test
 
 on:
   pull_request:
+    paths:
+    - 'otel-agent/k8s-helm/**'
```

Comment:
Could we maybe rename the file to `otel-agent-helm-test.yml` to keep it uniform?